src/Controller/NonPaginatedEntityIdListingControllerTrait.php

```diff
@@ -40,12 +41,13 @@ public function getEntityIds(): array
         ];
         $uri = $this->getBaseEndpointUri()->withQuery(http_build_query($query_params));
         $response = $this->getClient()->get($uri);
+        $expandCompatibility = (ClientInterface::HYBRID_ENDPOINT == $this->getClient()->getEndpoint());
```

Comment:
What if hybrid gets supported on on-premise installations as well? I mean, should we check rather the hybrid feature enabled on the org (`Apigee\Edge\Utility\OrganizationFeatures::isHybridEnabled()`) and loading the org for that, just like we do to[ figure out whether CPS is supported or not](https://github.com/apigee/apigee-client-php/blob/2.x-hybrid/src/Controller/PaginationHelperTrait.php#L134-L136), instead of relying on the endpoint?

Reply:
Hybrid's management API will always be called through the Apigee-hosted management plane (https://docs.apigee.com/hybrid/beta2/what-is-hybrid#api-programs-in-a-hybridmulti-cloud-world), so we can be sure this will not change. 
As for checking if the organization `isHybridEnabled()` or not, I agree that it would make for a more consistent use, however, it will:
- Add another call to the API when we can already deduce this from the client.
- Require that some controllers use `OrganizationControllerAwareTrait` and `OrganizationAwareControllerTrait`. The current pattern of using these traits is to make the controllers have an `$organizationController` and `$organization` properties, which can be injected through constructors. Changing constructors would then be a breaking change, because even if we can set a default value for `$organizationController`, we can't do that for `$organization`.
So overall I still think checking the client endpoint works for this case. What do you think?